src/index.css

```diff
@@ -1,13 +1,129 @@
 body {
-  margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
-    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
+  margin: 0 auto;
+  padding: 10px 15px;
+  font-family: sans-serif;
 }
 
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
-    monospace;
+.title {
+  display: grid;
+  width: 90%;
+  margin: 0 auto;
+  grid-template-columns: 1fr 1fr;
+}
+
+.btn {
+  border-radius: 5px;
+  font-size: 20px;
+  border: none;
+  color: #343837;
+  background-color: #96cac1;
+}
+
+button:hover,
+button:focus {
+  outline: none;
+}
+
+h1 {
+  text-align: center;
+}
+
+.btn-head {
+  display: flex;
+  margin-top: 5px;
+  padding: 2px;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.wrapper {
```

Comment:
- [ ] The styling of the list on your Homepage does not conform to the alternating dual-color pattern of the provided [design template](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), and therefore difficult to differentiate the details of each list item. Kindly note we as much as possible want your design to match the provided [template](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)). Therefore kindly ensure that cells of your list follow the **alternating dual-color pattern**. Each cell of the list item should have surrounding cells having **different (alternating) background color(s)**.
- [ ] Kindly ensure that data is properly wrapped within the cells/ _divs_ on the _display page_.

**Your design:**
![image](https://user-images.githubusercontent.com/12745474/149150166-f7d32fae-439b-4ad3-9fe4-3bf7ca7ced47.png)

![image](https://user-images.githubusercontent.com/12745474/149151332-73df2af9-a4a7-4d15-95cb-4821fcece0b0.png)



**The original design:**
![image](https://user-images.githubusercontent.com/12745474/142270463-4d16b9d3-7557-430c-80eb-a7c40589a4a6.png)

![image](https://user-images.githubusercontent.com/12745474/149150871-e9385b03-84c3-4285-82ed-d12943e24204.png)